cmd/frontend/internal/app/ui/raw.go

```diff
@@ -237,26 +260,25 @@ func serveRaw(w http.ResponseWriter, r *http.Request) error {
 				names = append(names, name)
 			}
 			result := strings.Join(names, "\n")
-			log15.Debug("raw endpoint sending file", "repo", common.Repo.Name, "commit", common.CommitID, "path", requestedPath, "type", "dir", "size", len(infos))
-			metricRawTotal.WithLabelValues("dir").Inc()
 			fmt.Fprintf(w, "%s", template.HTMLEscapeString(result))
 			return nil
 		}
 
 		// File
+		requestType = "file"
+		size = fi.Size()
 		f, err := archiveFS.Open(r.Context(), requestedPath)
 		if err != nil {
 			return err
 		}
 		defer f.Close()
-		log15.Debug("raw endpoint sending file", "repo", common.Repo.Name, "commit", common.CommitID, "path", requestedPath, "type", "file", "size", fi.Size())
-		metricRawTotal.WithLabelValues("file").Inc()
 		_, err = io.Copy(w, f)
 		return err
 	}
 }
 
-var metricRawTotal = promauto.NewCounterVec(prometheus.CounterOpts{
-	Name: "src_http_raw_endpoint_total",
-	Help: "The total number of requests to the raw endpoint API.",
-}, []string{"type"})
+var metricRawDuration = promauto.NewHistogramVec(prometheus.HistogramOpts{
+	Name:    "src_http_raw_duration_seconds",
```

Comment:
I think this is the convention?
```suggestion
	Namespace: "src",
    Subsystem: "frontend",
	Name:      "http_raw_duration_seconds",
```

Reply:
that is a relic of the past :) Name is more succinct and greppable.

Reply:
OK... but then this is very inconsistent and `http_raw` is pretty vague, we should at least have "frontend" in it, i.e. "src_frontend_http_raw_duration_seconds".

FYI we also have https://github.com/sourcegraph/promgrep to grep for Prometheus metrics.